syntax = "proto3";

package small.server_reg;

service ServerReg {
  rpc Register(ServerRegRequest) returns (ServerRegReply) {}
}

message ServerRegRequest {
  string sql_address = 1;
  string rpc_address = 2;
  string region = 3;
}

message ServerRegReply {
  bool success = 1;
}
